src/careamics/config/support/supported_data.py

```diff
@@ -82,3 +82,28 @@ def get_extension(cls, data_type: Union[str, SupportedData]) -> str:
             return "*.*"
         else:
             raise ValueError(f"Data type {data_type} is not supported.")
+
+    @classmethod
+    def get_extension(cls, data_type: Union[str | SupportedData]) -> str:
+        """
+        Get file extension of corresponding data type.
+
+        Parameters
+        ----------
+        data_type : str or SupportedData
+            Data type.
+
+        Returns
+        -------
+        str
+            Corresponding extension.
+        """
+        if data_type == cls.ARRAY:
+            raise NotImplementedError(f"Data {data_type} are not loaded from file.")
+        elif data_type == cls.TIFF:
+            return ".tiff"
+        elif data_type == cls.CUSTOM:
+            # TODO: improve this message
+            raise NotImplementedError("Custom extensions have to be passed elsewhere.")
```

Comment:
Can we give more details in the error message? Right now I am not sure yet where this method is called, but we are pretty consistent about the `extension_filter` parameter for the `get_extension_pattern` method.

Reply:
ah nevermind, there is todo!

Reply:
Yes this function isn't called anywhere yet, so I decided to leave the message as a future job, especially because there might be some refactoring. 

In future it will be used to add the correct extension when saving predictions. It made it into this PR because I was considering putting it directly into the `write_tiff` function by doing something like `file_path.with_suffix(SupportedData.get_extension(data_type))`. But then I decided against it, and instead thought file paths should be correct before being passed to the save function, because of how custom save functions will be called.